app/helpers/spree/api/api_helpers.rb

```diff
@@ -86,7 +86,7 @@ def state_attributes
       end
 
       def adjustment_attributes
-        [:id, :source_type, :source_id, :adjustable_type, :adjustable_id, :originator_type,
+        [:id, :adjustable_type, :adjustable_id, :originator_type,
```

Comment:
Unless I'm missing some metaprogramming somewhere, it looks like almost everything in this class is dead code, Spree leftovers that aren't used any more. I'll check it out...

Reply:
https://github.com/openfoodfoundation/openfoodnetwork/pull/7352

:arrow_right: :wastebasket: